lib/shared-ini/ini-loader.js

```diff
@@ -0,0 +1,70 @@
+var AWS = require('../core');
+var os = require('os');
+var path = require('path');
+
+module.exports = AWS.util.inherit({
```

Comment:
If you want this to appear in documentation, you also need to attach IniLoader to the AWS namespace:
```javascript
AWS.IniLoader = AWS.util.inherit/* ... */

// optionally also export it:
module.exports = AWS.IniLoader;
```

You'll also want to add doc strings to the public methods.